mlflow/server/handlers.py

```diff
@@ -344,12 +348,6 @@ def _get_artifact_repo(run):
     if run.info.artifact_uri:
         return ArtifactRepository.from_artifact_uri(run.info.artifact_uri, store)
 
-    # TODO(aaron) Remove this once everyone locally only has runs from after
```

Comment:
Thanks for the cleanup. We should remove the `if` statement (or maybe turn it into an `assert` to make it clear that the run does not have an artifact URI), otherwise we have a path with no `return`.

Reply:
Good catch. Thanks